### PR TITLE
TE923: Fix issue #574 with 1/36 inch rain counter.

### PR DESCRIPTION
**Pull request by twitham1**
See also issue #574 for more details and examples of why I believe rain counter should report 1/36 inch to match hardware console.  Fixing the driver at the source seems better than all users having to add a correction.  Unless, of course, more users can show that the current value is working as it should.  Will ask them on weewx-users.

### CODE REVIEW

File: bin/weewx/drivers/te923.py [hunk 1]
```diff
@@ -562,7 +566,7 @@ class TE923Configurator(weewx.drivers.AbstractConfigurator):
     }
     
     city_dict = {
-        0: ["ADD", 3, 0, 9, 0o1, "N", 38, 44, "E", "Addis Ababa, Ethiopia"],
+        0: ["ADD", 3, 0, 9, 1, "N", 38, 44, "E", "Addis Ababa, Ethiopia"],
```
Comment (twitham1) [inline bin/weewx/drivers/te923.py:569]: This is unrelated cleanup of automated 01 -> 0o1 octal change, better stated as simply 1.  Zero functionality change here.

**Comment by tkeffer** [comment]
Let's give it a try. I also made an entry in the change log. https://github.com/weewx/weewx/commit/4beed401cd9ce8882f53833dc7267af56461d5be

**Comment by twitham1** [comment]
Thank you Tom!

It turns out my new comment at line 2035 was not needed and can be removed or changed to document that the units reported there are millimeters.

That wee_device --minmax option reports the values that the console displays in the memory button.  I count 167 bucket tips in my database for the month of May and the console shows 4.64 inches, exactly matching the 1/36" records I have been keeping.  1/36*167=4.6388888, matching the hardware console!  This will be correctly reported as 118 millimeters by --minmax with the new multiplier.

**Comment by tkeffer** [comment]
I wonder why on earth they chose 1/36" as a bucket size...?

**Comment by twitham1** [comment]
I wonder if the geometry of the hardware just happened to generate 36 tips per inch when it was calibrated?  If so, then why on earth did they call this 0.03 inch or 0.6578 mm in the manual?  This documentation bug has caused many wrong measurements over the years.
